share/locale/en.json

```diff
@@ -1,165 +1,165 @@
 {
-    "config_onion_service": "Configuring onion service on port {0:d}.",
-    "preparing_files": "Preparing files to share.",
+    "config_onion_service": "Setting up onion service on port {0:d}.",
+    "preparing_files": "Preparing OnionShare.",
     "wait_for_hs": "Waiting for HS to be ready:",
     "wait_for_hs_trying": "Trying…",
     "wait_for_hs_nope": "Not ready yet.",
-    "wait_for_hs_yup": "Ready!",
-    "give_this_url": "Give this address to the person you're sending the file to:",
-    "give_this_url_stealth": "Give this address and HidServAuth line to the person you're sending the file to:",
-    "give_this_url_receive": "Give this address to the people sending you files:",
-    "give_this_url_receive_stealth": "Give this address and HidServAuth line to the people sending you files:",
+    "wait_for_hs_yup": "Ready.",
+    "give_this_url": "Give this address to the recipient of the file:",
+    "give_this_url_stealth": "Give this address and HidServAuth line to the recipient of the file:",
+    "give_this_url_receive": "Give this address to those that send you files:",
+    "give_this_url_receive_stealth": "Give this address and your HidServAuth line to those that send you files:",
     "ctrlc_to_stop":  "Press Ctrl+C to stop the server",
     "not_a_file": "{0:s} is not a valid file.",
     "not_a_readable_file": "{0:s} is not a readable file.",
-    "no_filenames": "You must specify a list of files to share.",
-    "no_available_port": "Could not start the Onion service as there was no available port.",
-    "download_page_loaded": "Download page loaded",
-    "other_page_loaded": "Address loaded",
-    "close_on_timeout": "Stopped because timer expired",
-    "closing_automatically": "Stopped because download finished",
-    "timeout_download_still_running": "Waiting for download to complete",
+    "no_filenames": "Add files to OnionShare.",
+    "no_available_port": "Could not start the onion service without a port to do it on.",
+    "download_page_loaded": "OnionShare page loaded",
+    "other_page_loaded": "OnionShare address loaded",
+    "close_on_timeout": "Timer expiry stopped OnionShare",
+    "closing_automatically": "OnionShare completed, now stopped",
+    "timeout_download_still_running": "Awaiting OnionShare completion",
```

Comment:
For the previous two, I do kind of like keeping the word "download", specifically to differentiate it from "upload" which is something else that you can do in OnionShare now

Reply:
Fine point.
"Onionshare sent, now stopped"?
and
"Awaiting full OnionShare"?

Reply:
Download finished, now stopped
Awaiting completion of the download

These make more sense to me. OnionShare is an application: it's not the thing being shared.